osgar/node.py

```diff
@@ -0,0 +1,31 @@
+"""
+  Node - processing unit
+"""
+from threading import Thread
+
+from osgar.bus import BusShutdownException
+
+
+class Node(Thread):
+    """
+       Parent classs for all processing "nodes".
+    """
+    def __init__(self, config, bus):
+        super().__init__()
+        self.setDaemon(True)
+        self.bus = bus
+
+    def update(self):
+        timestamp, channel, data = self.bus.listen()
```

Comment:
This method does not do anything, right? I am confused. Should it even be here?

Reply:
it pulls data from queue ... and could update `self.time`

Reply:
Now I like it.